### DISCUSSION TITLE
第5节课笔记

### DISCUSSION THREAD

**Discussion by BarryYin** [InternLM/Tutorial#408] [笔记]
链接添加

**Comment by liuwei16** [comment]
https://g5b4uupmby.feishu.cn/wiki/QFEKwUNrfi4bXukLAL9cDwEvnOc

**Comment by mmb135** [comment]
https://www.yuque.com/r/note/a9d9d22b-33fc-4a8b-862e-399d99586833

**Comment by lankuohsing** [comment]
https://github.com/lankuohsing/InternLM_notes/blob/main/lecture_5/homework_5.md

**Comment by cavedweller509** [comment]
https://github.com/BestAnHongjun/InternLM_Notes/blob/main/notes/lecture_5.md

**Comment by funcfang** [comment]
https://www.yuque.com/funcfang/ke5qbq/bapf4rnfgor86xiz?singleDoc#

**Comment by CDL0726** [comment]
https://github.com/CDL0726/InternLM_Notes/blob/main/README.md

**Comment by Jasimyn** [comment]
https://broadleaf-gemini-274.notion.site/Lecture5-LMDeploy-9f0afb64fa404e59ac4241bde25d40da?pvs=4
